data/users/externalldap.py

```
@@ -365,9 +407,17 @@ def ping(self):
                 pass
         except ldap.INVALID_CREDENTIALS:
             return (False, "LDAP Admin dn or password is invalid")
+        except LDAP_CONNECTION_ERRORS as ldaperr:
+            args = _log_ldap_error(
+                ldaperr, details=True, additional="Exception when trying to health check LDAP"
+            )
+            return (False, str(args))
         except ldap.LDAPError as lde:
```

Ruff (0.14.7)

Local variable `lde` is assigned to but never used

Remove assignment to unused variable `lde`

(F841)